src/components/StatsPanel/StatsPanel.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { styled } from '@mui/material/styles';
+
+import type { PaletteColor } from '@mui/material/styles';
+
+export const CardContainer = styled('div')(({ theme }) => ({
+  backgroundColor: theme.palette.background.paper,
+  borderRadius: theme.spacing(2.5),
+  padding: theme.spacing(4),
+  display: 'flex',
+  flexDirection: 'column',
+  gap: theme.spacing(3),
+  width: theme.spacing(32.5),
+  boxShadow: theme.shadows[3],
+}));
+
+export const StatItem = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  gap: theme.spacing(1.75),
```

Comment:
This file uses fractional spacing values (theme.spacing(1.75)). Fractional spacing is allowed but inconsistent with typical design-system spacing scales (usually integer multipliers of theme.spacing). Using fractions can hinder consistency across components.

**Sugerencia:**
```typescript
Prefer the design system spacing scale (integers). Replace fractional values with the closest integer token e.g. theme.spacing(2) or adjust the design tokens if fractional spacing is truly required.
```